assets/src/admin/paired-browsing/app.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * WordPress dependencies
+ */
+import { removeQueryArgs, addQueryArgs, hasQueryArg } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import './app.css';
+
+const { ampSlug, ampPairedBrowsingQueryVar, alert, history } = window;
+
+class PairedBrowsingApp {
+	/**
+	 * Set the iframes on init.
+	 */
+	constructor() {
+		this.nonAmpIframe = document.getElementById( 'non-amp' ).firstElementChild;
+		this.ampIframe = document.getElementById( 'amp' ).firstElementChild;
+
+		Promise.all( this.getIframeLoadedPromises() ).then( () => {
+			// Check every second to see if any iframe has disconnected.
+			setInterval( () => this.checkConnectedIframes(), 1000 );
+
+			const exitLink = document.getElementById( 'exit-link' );
+
+			exitLink.addEventListener( 'click', () => {
```

Comment:
As mentioned below, we should make this accessible by using a proper anchor. Instead of using `window.location.replace` and a click handler we can update the link's `href` value with `this.nonAmpIframe.contentWindow.location.href` if they don't match (e.g. when navigating)

Reply:
Thanks for the guidance @swissspidy.